include/oneapi/tbb/task_arena.h

```diff
@@ -302,6 +305,11 @@ class task_arena : public task_arena_base {
         }
     }
 
+    //! Creates an instance of task_arena attached to the current arena of the thread
+    explicit task_arena(d1::attach)
```

Comment:
Can we avoid introduction of new overloads by changing the definition of `task_arena::atach` to be : 
```c++
using atach = d1::attach;
```

Reply:
It might break already compiled if it passes the `attach` object to new code (and vice versa)

Reply:
I was thinking that mixing object files compiled with different versions of oneTBB is not supported, or I missed something ?

Reply:
It is supported within the same namespace interface version

Reply:
it seems that spec require these methods https://spec.oneapi.io/versions/latest/elements/oneTBB/source/task_scheduler/task_arena/task_arena_cls.html 

And old version are declared [deprecated  ](https://spec.oneapi.io/versions/latest/elements/oneTBB/source/deprecated/task_arena_attach_tag.html), but not yet removed